tests/test_proto.py

```diff
@@ -12,3 +12,21 @@ def test_imports():
     from substrait.gen.proto.type_expressions_pb2 import DerivationExpression
     from substrait.gen.proto.type_pb2 import Type
     from substrait.gen.proto.extensions.extensions_pb2 import SimpleExtensionURI
+
```

Comment:
We can go ahead and delete the test_imports() function since the new test indirectly tests these imports.

Reply:
I think there is a value in keeping this test to ensure backward compatibility.
`substrait.proto` might continue to work but import the classes from modules with different names than the current ones. 

In such case we would be breaking backward compatibility for anyone directly accessing the classes from the `_pb2` modules and that's something we want to catch and be aware of.